New sample ace-my-notifications

joaojmendes opened the pull request:
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                              |
| New sample?     | yes                              |
 

## What's in this Pull Request?

New Sample ace-my-notifications
receive Realtime list notifications

PopWarner commented:
Thank you @joaojmendes! You rock! We will get this processed soon. 💜🧡

I'll also reach out directly about seeing if we can demo this. 👏🏻👏🏻

dcashpeterson commented:
@joaojmendes Thanks for the sample. I just merged it into the main branch. One thing, the assets folder ended up in the root so I moved it when I changed the folder name.

joaojmendes commented:
Thank you very much Derek 🙂🙂

Obter o Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Derek Cash-Peterson ***@***.***>
Enviado: Friday, December 10, 2021 1:39:21 PM
Para: pnp/sp-dev-fx-aces ***@***.***>
Cc: João Mendes ***@***.***>; Mention ***@***.***>
Assunto: Re: [pnp/sp-dev-fx-aces] New sample ace-my-notifications (PR #17)


@joaojmendes<https://github.com/joaojmendes> Thanks for the sample. I just merged it into the main branch. One thing, the assets folder ended up in the root so I moved it when I changed the folder name.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/pnp/sp-dev-fx-aces/pull/17#issuecomment-990982155>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AC7YHVLFGRTCAX7NKJ7PTNTUQH7ITANCNFSM5IZT2B2A>.
